schema.sql

```diff
@@ -0,0 +1,89 @@
+DROP TABLE IF EXISTS earthquakes;
+DROP TABLE IF EXISTS user_topic_assignment;
+DROP TABLE IF EXISTS topics;
+DROP TABLE IF EXISTS alerts;
+DROP TABLE IF EXISTS magnitude;
+DROP TABLE IF EXISTS networks;
+DROP TABLE IF EXISTS type;
+DROP TABLE IF EXISTS users;
+DROP TABLE IF EXISTS region;
+
+CREATE TABLE alerts (
+    alert_id SMALLINT GENERATED ALWAYS AS IDENTITY,
+    alert_type VARCHAR(6) UNIQUE NOT NULL,
+    PRIMARY KEY alert_id
+);
+
+CREATE TABLE magnitude (
+    magnitude_id SMALLINT GENERATED ALWAYS AS IDENTITY,
+    magnitude_type VARCHAR(3) UNIQUE NOT NULL,
+    PRIMARY KEY magnitude_id
+);
+
+CREATE TABLE networks (
+    network_id SMALLINT GENERATED ALWAYS AS IDENTITY,
+    networK_name VARCHAR(2) UNIQUE NOT NULL,
+    PRIMARY KEY network_id
+);
+
+CREATE TABLE type (
+    type_id SMALLINT GENERATED ALWAYS AS IDENTITY,
+    type_name VARCHAR(10) UNIQUE NOT NULL,
+    PRIMARY KEY type_id
+);
+
+CREATE TABLE earthquakes (
+    earthquake_id BIGINT GENERATED ALWAYS AS IDENTITY,
+    time TIMESTAMPTZ NOT NULL,
+    tsunami BOOLEAN,
+    felt_report_count SMALLINT,
+    cdi FLOAT,
+    latitude FLOAT NOT NULL,
+    longitude FLOAT NOT NULL,
+    detail_url VARCHAR(255) UNIQUE NOT NULL
+    alert_id SMALLINT,
+    magnitude_id SMALLINT,
+    network_id SMALLINT,
+    type_id SMALLINT,
+    PRIMARY KEY earthquake_id,
+    FOREIGN KEY alert_id REFERENCES alerts(alert_id),
+    FOREIGN KEY magnitude_id REFERENCES magnitude(magnitude_id),
+    FOREIGN KEY network_id REFERENCES networks(network_id),
+    FOREIGN KEY type_id REFERENCES type(type_id)
+);
+
+CREATE TABLE users (
+    user_id BIGINT GENERATED ALWAYS AS IDENTITY,
+    email VARCHAR(255) UNIQUE NOT NULL,
+    phone VARCHAR(15) UNIQUE NOT NULL,
+    PRIMARY KEY user_id
+);
+
+CREATE TABLE regions (
+    region_id BIGINT GENERATED ALWAYS AS IDENTITY,
+    min_latitude FLOAT NOT NULL,
```

Comment:
Using FLOAT for latitude and longitude is fine, but consider DECIMAL for greater precision maybe?